README.md

```diff
@@ -11,87 +14,76 @@ This emulator is currently in the alpha state, so while it is intended to be
 stable, it is possible that it could crash and cause data loss. It is advised
 to put any important files in Archive memory before running the emulator.
 
-Grab the latest pre-built releases at https://github.com/calc84maniac/tiboyce/releases
+Grab the latest pre-built releases at <https://github.com/calc84maniac/tiboyce/releases>
 or check out the build instructions below to build from source.
 
 Features
 --------
 
-* Emulates original Game Boy hardware (except audio and linking)
-* Emulates real-time clock for certain cartridges
-* Save states with compression
-* Fullscreen and 1:1 scaling modes (with optional skin)
-* Automatic and manual frameskip
-* Turbo mode (with speed display)
-* GBC-style selectable color palettes for Game Boy games
-* Customizable controls
+*   Emulates original Game Boy hardware (except audio and linking)
+*   Emulates real-time clock for certain cartridges
+*   Save states with compression
+*   Fullscreen and 1:1 scaling modes (with optional skin)
+*   Automatic and manual frameskip
+*   Turbo mode (with speed display)
+*   GBC-style selectable color palettes for Game Boy games
+*   Customizable controls
 
 Converting ROM files
 --------------------
 
 The easiest method is to use the online converter [here](https://calc84maniac.github.io/tiboyce/converter).
 All conversion is done locally in your browser, so there's no need to worry about uploading copyrighted data.
 
-Alternatively, a command-line utility (tiboyce-romgen.exe) to convert ROM files
-to TI AppVars is included. There are a couple of ways to do this:
+Alternatively, a command-line utility (tiboyce-romgen.exe) to convert ROM files to TI AppVars is included. There are a couple of ways to do this:
 
-1. Open a command prompt and navigate to the directory containing the utility
-   using `cd`. Then run the following:
+1.  Open a command prompt and navigate to the directory containing the utility using `cd`. Then run the following:
```

Comment:
Looks like the following line (code) doesn't get formatted as such anymore. Probably should mark it explicitly with ```